README.md

```diff
@@ -1 +1,15 @@
-# joda-to-java-time-bridge
\ No newline at end of file
+# joda-to-java-time-bridge
+
+[![Build Status](https://travis-ci.org/airlift/joda-to-java-time-bridge.svg?branch=master)](https://travis-ci.org/airlift/joda-to-java-time-bridge)
+
+Mixed use of joda and java.time results in two different versions of
```

Comment:
Should we put backticks around `java.time`?

Reply:
Yes, and use the correct name Joda-Time